### ISSUE TITLE
Charging MX Ergo shows "0%" the entire time while recharging

### ISSUE THREAD

**Issue by khaytsus** [pwr-Solaar/Solaar#682]
**Information**

- Solaar version:  1.0.1
- Distribution: Fedora 30
- Kernel version (ex. `uname -srmo`): Linux 5.4.18-100.fc30.x86_64 x86_64 GNU/Linux

- Output of `solaar show`:
```
Unifying Receiver
  Device path  : /dev/hidraw5
  USB id       : 046d:c52b
  Serial       : A027983A
    Firmware   : 12.09.B0030
    Bootloader : 04.16
    Other      : AA.AA
  Has 1 paired device(s) out of a maximum of 6.
  Notifications: wireless, software present (0x000900)
  Device activity counters: 1=181

  1: MX Ergo Multi-Device Trackball 
     Codename     : MX Ergo
     Kind         : trackball
     Wireless PID : 406F
     Protocol     : HID++ 4.5
     Polling rate : 8 ms (125Hz)
     Serial number: BD87365D
        Bootloader: BOT 49.00.B0001
          Firmware: MPM 06.03.B0022
             Other: 
     The power switch is located on the base.
     Supports 36 HID++ 2.0 features:
         0: ROOT                   {0000}   
         1: FEATURE SET            {0001}   
         2: DEVICE FW VERSION      {0003}   
         3: DEVICE NAME            {0005}   
         4: WIRELESS DEVICE STATUS {1D4B}   
         5: unknown:0007           {0007}   
         6: RESET                  {0020}   
         7: unknown:0021           {0021}   
         8: BATTERY STATUS         {1000}   
         9: LED CONTROL            {1300}   
        10: REPROG CONTROLS V4     {1B04}   
        11: unknown:1C00           {1C00}   
        12: unknown:2006           {2006}   
        13: POINTER SPEED          {2205}   
            Pointer Speed: 1.0
        14: VERTICAL SCROLLING     {2100}   
            Roller type: standard
            Ratchet per turn: 18
            Scroll lines: 0
        15: unknown:00C2           {00C2}   
        16: unknown:1802           {1802}   internal, hidden
        17: unknown:1803           {1803}   internal, hidden
        18: unknown:1806           {1806}   internal, hidden
        19: unknown:1805           {1805}   internal, hidden
        20: unknown:1813           {1813}   internal, hidden
        21: CHANGE HOST            {1814}   
        22: unknown:1815           {1815}   
        23: unknown:1830           {1830}   internal, hidden
        24: unknown:1861           {1861}   internal, hidden
        25: unknown:1890           {1890}   internal, hidden
        26: unknown:1891           {1891}   internal, hidden
        27: unknown:18A1           {18A1}   internal, hidden
        28: unknown:1DF3           {1DF3}   internal, hidden
        29: unknown:1E00           {1E00}   hidden
        30: unknown:1EB0           {1EB0}   internal, hidden
        31: unknown:18B1           {18B1}   internal, hidden
        32: unknown:1850           {1850}   internal, hidden
        33: unknown:1F03           {1F03}   internal, hidden
        34: unknown:18C0           {18C0}   internal, hidden
        35: LOWRES WHEEL           {2130}   
            Wheel Reports: HID
     Has 9 reprogrammable keys:
         0: LEFT CLICK                , default: LeftClick                   => LEFT CLICK                
             mse, reprogrammable, pos:0, group:1, gmask:1
         1: RIGHT CLICK               , default: RightClick                  => RIGHT CLICK               
             mse, reprogrammable, pos:0, group:1, gmask:1
         2: MIDDLE BUTTON             , default: MiddleMouseButton           => MIDDLE BUTTON             
             mse, reprogrammable, divertable, persistently divertable, pos:0, group:2, gmask:3
         3: BACK AS BUTTON 4          , default: BackEx                      => BACK AS BUTTON 4          
             mse, reprogrammable, divertable, persistently divertable, pos:0, group:2, gmask:3
         4: FORWARD AS BUTTON 5       , default: BrowserForwardEx            => FORWARD AS BUTTON 5       
             mse, reprogrammable, divertable, persistently divertable, pos:0, group:2, gmask:3
         5: unknown:00ED              , default: unknown:00C4                => unknown:00ED              
             mse, reprogrammable, divertable, persistently divertable, pos:0, group:2, gmask:3
         6: LEFT SCROLL AS AC PAN     , default: HorzScrollLeftSet           => LEFT SCROLL AS AC PAN     
             mse, reprogrammable, divertable, persistently divertable, pos:0, group:2, gmask:3
         7: RIGHT SCROLL AS AC PAN    , default: HorzScrollRightSet          => RIGHT SCROLL AS AC PAN    
             mse, reprogrammable, divertable, persistently divertable, pos:0, group:2, gmask:3
         8: unknown:00D7              , default: unknown:00B4                => unknown:00D7              
             divertable, virtual, pos:0, group:3, gmask:0
     Battery: 0%, recharging.
```

**Describe the bug**
The entire time while the MX Ergo is charging, solaar show just shows "Battery: 0%, recharging." for the battery state.  It's obviously not a huge deal as I just unplug the cable when I see the green light on, but seems odd.  It could be that the real battery level isn't indicated though while charging.  If there's anything I can do to capture more data there, please let me know.

Also as a side note; it also only ever shows 20%, 50%, and 90% when not charging.  I'll log this separately.

**To Reproduce**
1. Charge the MX Ergo from any state (except full I suppose?)
2. Watch solaar show
3. See error

**Comment by pfps** [comment]
This should have been fixed by PR #668.  To test, clone the Solaar from this repository and run it.

**Comment by khaytsus** [comment]
Yep sorry I missed that existing bug in the PR, indeed I see "Battery: N/A, recharging." now which is less confusing.  Thank you!
